api/api/utils/image_proxy/extension.py

```diff
@@ -37,7 +49,12 @@ async def get_image_extension(image_url: str, media_identifier) -> str | None:
             else:
                 ext = None
 
-            do_not_wait_for(sync_to_async(cache.set)(key, ext if ext else "unknown"))
+            if is_redis_reachable:
+                do_not_wait_for(
+                    sync_to_async(cache.set)(key, ext if ext else "unknown")
+                )
+            else:
+                logger.warning("Redis connect failed, cannot cache image extension.")
```

Comment:
It does say the opposite thing, one was for being unable to read from the cache and the other is for being unable to write to it. The `is_redis_reachable` variable is just to avoid a potential network call to a service that's not accepting connections.